packages/react/src/css/reset.css

```diff
@@ -70,3 +70,7 @@ select {
     scroll-behavior: auto;
   }
 }
+
+html {
+  scroll-behavior: smooth;
+}
```

Comment:
@danielguillan I can't remember if you added this or I did 😅. Either way, do we still need this? If yes, it needs a changeset. If it's just for docs, we should move to Doctocat instead and not here.